src/api/request.ts

```diff
@@ -132,11 +141,14 @@ class Request extends Thenable<RequestResponse> implements RequestInterface {
   protected readonly promise: Promise<RequestResponse>
   protected readonly options: RequestOptions
   private readonly cancelController: CancelTokenSource
+  private throttledTimes: number = 0
+  private readonly retryThrottledMaxTimes: number
 
-  constructor(options: RequestOptions) {
+  constructor(options: RequestOptions, retryThrottledMaxTimes) {
```

Comment:
Why not to put it inside options?

Reply:
Because this is not an option of axios request as an example. And I do not know whether it should be. I just want to put options to Axios and don't worry about other params that would be. And even if it would be not axios.

Reply:
I just a little think and maybe this is make sense to include this param as option